cxx/2slice.hxx

```
@@ -694,6 +696,11 @@ public:
     }
 
     void _drop() noexcept;
+    BoxedStr clone() const noexcept;
```

Clang (14.0.6)

[warning] function 'clone' should be marked [[nodiscard]]

(modernize-use-nodiscard)